go-controller/pkg/ovn/master.go

```diff
@@ -648,7 +649,12 @@ func (oc *Controller) addNodeAnnotations(node *kapi.Node, hostSubnets []*net.IPN
 		return fmt.Errorf("failed to marshal node %q annotation for subnet %s",
 			node.Name, util.JoinIPNets(hostSubnets, ","))
 	}
-	err = oc.kube.SetAnnotationsOnNode(node, nodeAnnotations)
+	err = utilwait.PollImmediate(100*time.Millisecond, 1*time.Second,
```

Comment:
It would be nice to define the interval and timeout values as constants so that that could be used as standard values in other situations.

Reply:
ack.